cibyl/models/ci/base/feature.py

```diff
@@ -0,0 +1,39 @@
+"""
```

Comment:
maybe cibyl/models/ci/product/feature.py ?
I'm simply not sure a feature is a pure CI/CD entity

Reply:
That's a good point, I'll move it